operator/molecule/cluster/prepare.yml

```diff
@@ -52,11 +52,13 @@
     - ServiceCatalogAPIServer
     - ServiceCatalogControllerManager
 
+  - name: Wait a couple of seconds
```

Comment:
Waiting for what? Is this purely the equivalent to a sleep? Why are we doing this? I'd change the name to say why we are waiting. "Wait a few seconds to allow Foo Bar Baz to have time to come up" or something more descriptive. Otherwise, I'd come in later and see this wondering why the heck we're slowing things down.